### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by rudijs** [trusch/kv#1]
Hi,

Just trying out exactly as the readme describes, I get this error:

```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
```

```
make release                                                                                                                                                                                       main
cargo build --release
    Updating crates.io index
error: failed to select a version for the requirement `clap = "=4.0.10"`
candidate versions found which didn't match: 3.2.23, 3.2.22, 3.2.21, ...
location searched: crates.io index
required by package `kv v0.1.0 (/home/rudi/projects/kv)`
make: *** [Makefile:9: target/release/kv] Error 101
```

I'm not a Rust developer.

```
rustc --version                                                                                                                                                                                    main
rustc 1.51.0 (2fd73fabe 2021-03-23)
```

Let me know if there's anything else I can provide to debug.

**Comment by trusch** [comment]
I guess your compiler is too old, but thanks for the report, I'll check

**Comment by trusch** [comment]
Yep, your compiler is too old but on the good side a simple `sudo apt update && sudo apt upgrade` should fix this since Ubuntu 22.04 comes with rustc 1.61 which should hopefully be recent enough.

If you installed rust with rustup, `rustup update` should be sufficient
